Onepunch/README.md

```diff
@@ -1,18 +1,38 @@
-# Onepunch
+# Onepunch by Saitama D. Luffy
 
 ## Screenshots
 
 ![OnepunchHome](./OnepunchHome.png)
 ![OnepunchFullPlayer](./OnepunchFullPlayer.png)
 
 ## More
-This is still an alpha project intended to redesign the app as much as possible.
-But it will take time as I do this as a hobby. Hope you will like it.
 
-A theme by Saitama D. Luffy
+#### Details
 
-PS: feedbacks will be much appreciated. Specially from windows users. I can't test
-in windows.
+>&nbsp;&nbsp;&nbsp;&nbsp;A project intended to redesign the Spotify app.As you know Saitama is only a "hero for fun", this is going to take a while even though he can finish an enemy with just one punch. I hope you'll like it! Suggestions will always help me to make it look better.
+So, feel free to drop your feedback.
 
-Contact: Telegram https://t.me/saitama_a
-                Twitter https://twitter.com/_saitama_0
+#### Changelogs
+
+> Wondering what is changed, then check this [file](./changelog.md).
+
+#### A little help
+>To enable this theme type this code in your terminal app :
+>
+>      $spicetify config current_theme Onepunch
+>        //For more details go to spicetify github homepage.
+
+>To change between **day and night mode** just use this code in your terminal app :
+>
+>      $spicetify config color_scheme SCHEME
+>       //Replace SCHEME by BASE/base or DARK/dark for DAY or NIGHT mode respectively.
+>       //Schemes are case-insensitive.
```

Comment:
I think you should provide a screenshot for the new color scheme and add it to the README